index.html

```diff
@@ -36,11 +36,11 @@ <h2>Filter Starred Ideas</h2>
           <header>
               <button id="favorite" class="favorite-button">
                 <img name="star-white" src="assets/star.svg" alt="star">
-                <img name="star-red" src="assets/star-active.svg" alt="star">
+                <img name="star-red" id="starRed" src="assets/star-active.svg" alt="star">
               </button>
-              <button id="deleteCard" class="favorite-button">
+              <button id="deleteCard" class="delete-button">
```

Comment:
We should double check this button on the CSS file to make sure we did the change in every place where this information is being accessed `  <button id="deleteCard" class="delete-button">`